operations/network/management_network/mellanox/requirements_and_optional__configuration.md

```diff
@@ -0,0 +1,23 @@
+# How to connect management network to your campus network
```

Comment:
The new file name contains a excess underscore (requirements_and_optional__configuration.md), so it is treated as a new file, instead of renaming.

Reply:
Fixed and removed duplicate.